modules/ui/sections/color_selection.js

```diff
@@ -0,0 +1,91 @@
+import { uiTooltip } from '../tooltip';
+import { uiCombobox } from '../combobox';
+import { uiIcon } from '../icon';
+import { uiSection } from '../section';
+import { utilNoAuto } from '../../util';
+
+
+export function uiSectionColorSelection(context) {
+  const l10n = context.systems.l10n;
+  const colors = context.systems.colors;
+  colors.on('colorsloaded', loadComboBoxData);
+  let comboData = [];
+
+  function loadComboBoxData(){
+    let colorSchemeKeys = Object.keys(colors.getAllColorSchemes());
+
+    for (let i = 0; i < colorSchemeKeys.length; i++) {
+      let colorObject = {};
+      let colorSchemeTitle = colorSchemeKeys[i];
+      colorObject.title = colorSchemeTitle;
+      colorObject.value = colorSchemeTitle;
```

Comment:
This function here is where you should put your string resource fetch: I believe it is the `values` that are actually displayed by the comboBox code, so change the `value` of each object to essentially the same l10n call you have further down. 

```colorObject.value = l10n.t(`preferences.color_selection.${colorSchemeTitle}`);```

Reply:
wunderbar

<img width="374" alt="Screenshot 2023-12-04 at 12 11 42 PM" src="https://github.com/facebook/Rapid/assets/95086147/208e7474-5633-495c-b150-bccb1d2e6a7c">